src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+#block-container {
+  display: flex;
+  flex-direction: column;
+}
+
+#block-1 {
+  background-color: red;
+  width: 100%;
+  height: 100px;
+}
+
+#block-2 {
+  background-color: green;
+  width: 100%;
+  height: 100px;
+}
+
+#block-3 {
+  background-color: blue;
+  width: 100%;
+  height: 100px;
+}
```

Comment:
- all them have similar property, which can be moved to one general class and this class can be added to all the elements that need this style